website/docs/package.json

```diff
@@ -4,12 +4,13 @@
     "license": "MIT",
     "private": true,
     "scripts": {
-        "build": "run-s build:docusaurus",
-        "build:docusaurus": "docusaurus build --out-dir ../build",
+        "build": "run-s build:docusaurus build:copy",
+        "build:copy": "cp -r ./build ../build",
```

Comment:
Copying the directory for now. We can remove this step after another authentik release